app/views/users/index.html.erb

```diff
@@ -2,18 +2,17 @@
   <h3>Welcome to Blog App</h3>
 </div>
 
-<% @users.each do |user| %>
-  <%= link_to user do %>
-    <div>
-      <div class=box>
-        <img src="<%= user.photo%>", class: 'img'>
-        <div class=user-box>
-          <%= user.name %><br>
-          <div class=right>
-          number of posts: <%= user.posts_counter %><br>
+  <% @users.each do |user| %>
+    <%= link_to user do %>
+
+        <div class=box>
+          <img src="<%= user.photo%>", class: 'img'>
+          <div class=user-box>
+            <%= user.name %><br>
+            <div class=right>
+            number of posts: <%= user.posts_counter %><br>
+            </div>
           </div>
         </div>
-      </div>
-    </div>
-  <% end %>
```

Comment:
- [Optional] To avoid using a piece of code over and over, it is a good practice to use Rails partials. It renders your codebase DRY.